frontend/lib/activation/deeplink_activation.dart

```diff
@@ -8,15 +8,17 @@ import 'package:ehrenamtskarte/identification/id_card/id_card_with_region_query.
 import 'package:ehrenamtskarte/identification/qr_code_scanner/qr_code_processor.dart';
 import 'package:ehrenamtskarte/identification/user_code_model.dart';
 import 'package:ehrenamtskarte/identification/util/activate_card.dart';
+import 'package:ehrenamtskarte/identification/verification_workflow/verification_qr_code_processor.dart';
 import 'package:ehrenamtskarte/l10n/translations.g.dart';
 import 'package:ehrenamtskarte/proto/card.pb.dart';
 import 'package:flutter/material.dart';
 import 'package:go_router/go_router.dart';
+import 'package:intl/intl.dart';
 import 'package:provider/provider.dart';
 
 enum DeepLinkActivationStatus {
   /// Link is invalid
-  invalidLink,
+  invalidCardInfo,
```

Comment:
:upside_down_face:  I think there are more reasons why the activation could be invalid, not just due to invalid card infos
``` suggestion
// Activation is invalid
invalid,
```

Reply:
@f1sh1918 what do you think?

Reply:
i think its a valid suggestion, since we also have a fallback message if an unknown error happens